python/python/ert/ecl/ecl_grav.py

```diff
@@ -67,45 +66,49 @@ def __init__( self, grid, init_file ):
         c_ptr = self._grav_alloc( grid , init_file )
         super(EclGrav , self).__init__( c_ptr )
         
+        self.dispatch = {"FIP"    : self.add_survey_FIP,
+                         "RFIP"   : self.add_survey_RFIP,
+                         "PORMOD" : self.add_survey_PORMOD,
+                         "RPORV"  : self.add_survey_RPORV}
 
 
-    def add_survey_RPORV( self, survey_name, restart_file ):
+    def add_survey_RPORV( self, survey_name, restart_view ):
         """
         Add new survey based on RPORV keyword.
 
         Add a new survey; in this context a survey is the state of
         reservoir, i.e. an ECLIPSE restart file. The @survey_name
         input argument will be used when refering to this survey at a
-        later stage. The @restart_file input argument should be an
+        later stage. The @restart_view input argument should be an
         EclFile instance with data from one report step. A typical way
-        to load the @restart_file argument is:
+        to load the @restart_view argument is:
 
            import datetime
            import ert.ecl.ecl as ecl
            ...
            ...
            date = datetime.datetime( year , month , day )
-           restart_file1 = ecl.EclFile.restart_block( "ECLIPSE.UNRST" , dtime = date)
-           restart_file2 = ecl.EclFile.restart_block( "ECLIPSE.UNRST" , report_step = 67 )
+           restart_view1 = ecl.EclFile.restart_block( "ECLIPSE.UNRST" , dtime = date)
```

Comment:
`EclFile.restart_block` is deprecated and this should be updated to the new way of getting views.